events/repostream.go

```diff
@@ -36,20 +36,23 @@ func ConsumeRepoStreamLite(ctx context.Context, con *websocket.Conn, cb LiteStre
 					if err != nil {
 						e := fmt.Errorf("getting record %s (%s) within seq %d for %s: %w", op.Path, *op.Cid, evt.Seq, evt.Repo, err)
 						log.Error(e)
-						return nil
+						continue
 					}
 
 					if lexutil.LexLink(rc) != *op.Cid {
+						// TODO: do we even error here?
```

Comment:
I would say we should error out here. If the ops metadata is inconsistent with repo slice, that is pretty bad and we should error out and not partially process.

Maybe even mark repo as "dirty"? Something to think about this week.

Reply:
yeah, youre probably right